c2corg_api/models/report.py

```diff
@@ -0,0 +1,224 @@
+from c2corg_api.models.schema_utils import restrict_schema, get_update_schema
+from c2corg_common.fields_report import fields_report
+from sqlalchemy import (
+    Column,
+    Integer,
+    Boolean,
+    SmallInteger,
+    ForeignKey,
+    String,
+    Date
+    )
+
+from colanderalchemy import SQLAlchemySchemaNode
+
+from c2corg_api.models import schema, Base
+from c2corg_api.models.utils import ArrayOfEnum
+from c2corg_api.models.utils import copy_attributes
+from c2corg_api.models.document import (
+    ArchiveDocument,
+    Document,
+    geometry_schema_overrides,
+    schema_document_locale,
+    schema_attributes)
+from c2corg_common import document_types
+from c2corg_api.models import enums
+
+REPORT_TYPE = document_types.REPORT_TYPE
+
+
+class _ReportMixin(object):
+    #
+    # Caractérisation de l'évènement - Event characterization
+    #
+    longitude = Column(String)
+    latitude = Column(String)
```

Comment:
`longitute` and `latitude` are no explicit fields. It's handled with the geometry like for the other document types.